docs/src/build-from-source.md

```diff
@@ -30,6 +31,7 @@ git submodule update --init
 ./scripts/setup/ubuntu/install_deps.sh
 ./scripts/setup/ubuntu/install_cbmc.sh
 ./scripts/setup/ubuntu/install_viewer.sh
+./scripts/setup/install_kissat.sh
```

Comment:
Can we make the `install_deps.sh` to invoke all the other installation scripts? We can keep the other scripts since they are handy to update versions, but this list is starting to grow too much.

Reply:
I'll do this in a follow-up PR.